common/common.scss

```diff
@@ -1,182 +1,164 @@
-@import "badges_page";
+@import 'badgesPage';
```

Comment:
Updated for a former PR comment.
https://github.com/tidb-incubator/discourse-theme-asktug/pull/2#pullrequestreview-603826902